nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -103,7 +93,7 @@ public static void checkInstantiationForParameterizedTypedTree(
       return;
     }
     boolean[] typeParamsWithNullableUpperBound =
-        getTypeParamsWithNullableUpperBound(baseType, config, state, handler);
```

Comment:
Why do we no longer need 'state' here?

Reply:
It's because `hasNullableAnnotation` ((here)[https://github.com/uber/NullAway/pull/1142/files#diff-82d66c6ba15b87acc24509d7a3440bbe7606944197b929733d315fbe0a025435L1159-L1171]) used to rely on the `state` object in order to check for the presence of the JSpecify `@Nullable` annotation.  Now we don't do that, so in various places `state` is no longer needed (but `config` is).